src/codemods/colors-to-css-vars.ts

```diff
@@ -117,6 +120,9 @@ export default (file: FileInfo, api: API, options: Options) => {
     // Find Colors named imports in @freenow/wave imports
     const colorsImports = waveNamedImports.filter(path => path.node.imported.name === 'Colors');
 
+    // Early return in case colors is not used
+    if (colorsImports.size() === 0) return ast.toSource(printOptions);
```

Comment:
I need to change this to return `file.source` instead, so it does not even transform the code back from ast and returns the original source file instead

Reply:
Already done in #400